set.mm

```diff
@@ -465418,6 +465427,18 @@ its graph has a given second element (that is, function value).
 -.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
 $)
 
+  ${
+    $d M x $.
+    $( Upper integer sets are a subset of the corresponding closed-below,
+       open-above intervals.  (Contributed by Thierry Arnoux, 29-Dec-2021.) $)
+    uzssico $p |- ( M e. ZZ -> ( ZZ>= ` M ) C_ ( M [,) +oo ) ) $=
+      ( vx cz wcel cuz cfv cpnf cico co cv cle wbr wa cxr clt w3a ressxr sseldi
+      cr zssre sstri simprl simprr ltpnfd 3jca ex eluz1 wb sseli elico1 sylancl
+      pnfxr 3imtr4d ssrdv ) ACDZBAEFZAGHIZUOBJZCDZAURKLZMZURNDZUTURGOLZPZURUPDU
+      RUQDZUOVAVDUOVAMZVBUTVCVFCNURCSNTQUAZUOUSUTUBZRUOUSUTUCVFURVFCSURTVHRUDUE
+      UFAURUGUOANDGNDVEVDUHCNAVGUIULAGURUJUKUMUN $.
+  $}
+
```

Comment:
maybe proof can be shortened using ~elicopnf instead of ~elico1?
Maybe the proof of ~blennnelnn in my mathbox could be shortened by using ~uzssico? (To show that ( N e. NN -> N e. ( 1 [,) +oo ) ) is required).

Reply:
Indeed, this shortened the proof! Would you like me to mention you in the attribution?
I have not updated `~ blennnelnn` in your Mathbox, but this proof would certainly help shorten it.

Reply:
It's up to you if you want to mention me, I don't care.